python/libs/loris_api.py

```diff
@@ -0,0 +1,160 @@
+import json
+import requests
+import urllib
+
+class LorisAPI:
+    url = 'https://localhost/api/v0.0.4-dev/'
+    username = 'admin'
+    password = 'Pouliche123'
```

Comment:
Hi @laemtl, awesome work with all this cool stuff you've added! 

Eventually I'll try fixing the code for multiple windows of the GUI in react. I have some code for that commented out, it should already work but it was having problems when compiling to production. We could have a login button in the Configurations tab, with a similar position like the anonymize toggle you made but we can place the login button at the very top right (in the user's face) and where it opens a new window for user login credentials or maybe just an model like the loading functionality has. Then we could have three fields (url, username, password). So the EEG2BIDS tool can work on different LORIS instances if the user changes the URL.